tests/run_tests.py

```diff
@@ -27,23 +27,11 @@ def main(argv):
             continue
 
         print("+ [{}]".format(path))
-        run(
-            "cargo", "build", "-Zbuild-std=core,alloc",
-            "--target", target,
```

Comment:
I think we lost the target-generic-ness in the rebase. Do you want a `f'TARGET={target}'` or something? Deferring until #190?

Reply:
Yup, I don't see a way around that.